modules/andes-core/broker/src/main/java/org/wso2/andes/mqtt/MQTTLocalSubscription.java

```diff
@@ -33,6 +33,8 @@
 import java.nio.ByteBuffer;
 import java.util.UUID;
 
+import static org.dna.mqtt.wso2.AndesMQTTBridge.*;
```

Comment:
import grouping is there. I think there is a separate settings for static imports in intellij.